src/ResourceManager/Resources/Commands.Resources/Models.Authorization/AuthorizationClient.cs

```diff
@@ -59,13 +60,7 @@ static AuthorizationClient()
         public AuthorizationClient(IAzureContext context)
         {
             ActiveDirectoryClient = new ActiveDirectoryClient(context);
-            AuthorizationManagementClient = AzureSession.Instance.ClientFactory
-#if !NETSTANDARD
-                .CreateClient<AuthorizationManagementClient>
-#else
-                .CreateArmClient<AuthorizationManagementClient>
-#endif
-                (context, AzureEnvironment.Endpoint.ResourceManager);
+            AuthorizationManagementClient = AzureSession.Instance.ClientFactory.CreateArmClient<AuthorizationManagementClient>(context, AzureEnvironment.Endpoint.ResourceManager);
```

Comment:
>AuthorizationManagementClient [](start = 12, length = 29)

you might want to modify this file based on the content in the main branch,like
#if !NETSTANDARD
                .CreateClient<AuthorizationManagementClient>
#else
                .CreateArmClient<AuthorizationManagementClient>
#endif